src/utils/actions.js

```diff
@@ -401,9 +401,8 @@ export const fetchResponseHandler = (response) => {
     }
 }
 
-export const showMessage = (settings, callback = {}) => (dispatch) => {
+export const showMessage = (settings, callback) => (dispatch) => {
```

Comment:
callback should be declared as default value null ( callback = null) on signature